### PR TITLE
Nft & Crypto support ethereum

### PR DESCRIPTION
**Pull request by TalebRafiepour**
## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [x] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [x] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/dashboard/home/tab_bar/nft/cubit/nft_cubit.dart [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
 import 'package:equatable/equatable.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
 import 'package:json_annotation/json_annotation.dart';
 
 part 'nft_cubit.g.dart';
 
 part 'nft_state.dart';
 
-class NftCubit extends Cubit<NftState> {
+class NftCubit extends Cubit<NftState> with NFTCubitDao {
```
Comment (bibash28) [inline lib/dashboard/home/tab_bar/nft/cubit/nft_cubit.dart:17]: i am wondering how mixin is justified here...

**Comment by bibash28** [comment]
image
I think we should increase the precision to the 6th digit..The user has '0.003 Eth' but we are showing 0. The user may be confused. I think I increased precision in wallet connect transactions.

**Comment by bibash28** [comment]
When I pressed Eth in above screenshot.. i got this network error...
image

**Comment by bibash28** [comment]
We can also put dollar equivalent value of eth...

  static const ethPrice =
      'https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD';
      
image

**Comment by bibash28** [comment]
image

When I tried to transfer this token I faced following issue:
image

**Comment by bibash28** [comment]
Is token transfer totally free?

**Comment by TalebRafiepour** [comment]
@bibash28 OK Thanks for reviewing.
Adding USD value for ETH and another crypto on the Ethereum blockchain is another ticket.

**Pull request merged**